CorsixTH/Lua/languages/english.lua

```diff
@@ -1,4 +1,4 @@
---[[ Copyright (c) 2010-2020 Manuel "Roujin" Wolf, Edvin "Lego3" Linge, lewri et al.
+--[[ Copyright (c) 2010-2024 Manuel "Roujin" Wolf, Edvin "Lego3" Linge, MandrakeHorse, lewri et al.
```

Comment:
Not you but this copyright has taken a unique turn to our other normal formats. I'll try to remember to fix it up in future.